File: app/src/main/java/com/reed/reedplayer/utils/CheckUtils.java
package com.reed.reedplayer.utils;

import java.util.Collection;

/**
 * Created by thinkreed on 16/5/23.
 */
public class CheckUtils {

  public static <T> T get(T value, T defaultValue) {
    return value != null ? value : defaultValue;
  }

  public static boolean isEmpty(Collection collection) {
    return collection == null || collection.size() == 0;
  }
}

File: app/src/main/java/com/reed/reedplayer/utils/CheckUtils.kt
package com.reed.reedplayer.utils

/**
 * Created by thinkreed on 16/5/23.
 */
object CheckUtils {

    operator fun <T> get(value: T?, defaultValue: T): T {
        return value ?: defaultValue
    }

    fun isEmpty(collection: Collection<*>?): Boolean {
        return collection == null || collection.isEmpty()
    }
}
